### PR TITLE
Add ENV processor example

### PR DESCRIPTION
**Pull request by antoniovj1**
Added example of autowiring with env processor

### CODE REVIEW

File: service_container/autowiring.rst [hunk 1]
```diff
@@ -635,11 +635,15 @@ The ``#[Autowire]`` attribute can also be used for :ref:`parameters <service-par
...
+            string $mailerMethod,
 
             // environment variables
             #[Autowire(env: 'SOME_ENV_VAR')]
-            string $senderName
+            string $senderName,
+
+            // environment variables with processors
+            #[Autowire(env: 'bool:SOME_BOOL_ENV_VAR')]
+            string $allowAttachments,
```
Comment (OskarStark) [inline service_container/autowiring.rst:647]:
```suggestion
            bool $allowAttachments,
```
?

**Comment by OskarStark** [comment]
Thank you @antoniovj1.

File: service_container/autowiring.rst (hunk 1 above)
Comment (OskarStark) [reply service_container/autowiring.rst:647]: Fixed while merging

Comment (antoniovj1) [reply service_container/autowiring.rst:647]: Wow, what a silly mistake. Thank you very much!

**Pull request merged**